TFT/src/User/Menu/Probe.c

```diff
@@ -0,0 +1,51 @@
+#include "probe.h"
+#include "includes.h"
+
+const MENUITEMS probeItems = {
+// title
+LABEL_PROBE,
+// icon                        label
+ {{ICON_BLTOUCH_DEPLOY,        LABEL_BLTOUCH_DEPLOY},
+  {ICON_BLTOUCH_STOW,          LABEL_BLTOUCH_STOW},
+  {ICON_BLTOUCH_TEST,          LABEL_BLTOUCH_TEST},
+  {ICON_BLTOUCH_REPEAT,        LABEL_BLTOUCH_REPEAT},
+  {ICON_TOUCHMI,               LABEL_TOUCHMI_INIT},
+  {ICON_TOUCHMI,               LABEL_TOUCHMI_ZTEST},
+  {ICON_BACKGROUND,            LABEL_BACKGROUND},
+  {ICON_BACK,                  LABEL_BACK},}
+};
```

Comment:
why are both probes being displayed in the menu? There is only one of them is installed on the machines.

Reply:
> why are both probes being displayed in the menu?

I agree. Doubling up displayed probe types doesn't make sense.